### PR TITLE
#1386 fix nextInt and nextLong with extreme boundaries

### PR DESCRIPTION
**Pull request by asolntsev**
e.g.
* `randomService.nextInt(Integer.MIN_VALUE, Integer.MAX_VALUE)`
* `randomService.nextLong(Long.MIN_VALUE, Long.MAX_VALUE)`

### CODE REVIEW

**Comment by kingthorin** [comment]
Drop the unrelated workflow change?

**Comment by kingthorin** [comment]
To be clear I do think that's probably a handy change if you want to submit it separately.

(I'll leave that approval/merge up to bodiam but for my two cents it's good)

File: src/main/java/net/datafaker/providers/base/DateAndTime.java [hunk 1]
```diff
@@ -322,8 +322,8 @@ public Duration duration(long max, ChronoUnit unit) {
     /**
      * Generates a random Duration between min and max.
      *
-     * @param min  the maximum value
-     * @param max  the minimal value
+     * @param min  the minimum value
+     * @param max  the maximum value
```
Comment (kingthorin) [inline src/main/java/net/datafaker/providers/base/DateAndTime.java:326]: Ha good catch, one of obviously didn't notice the mismatch.

**Review by kingthorin — APPROVED** [review]

**Comment by asolntsev** [comment]
> that's probably a handy change if you want to submit it separately

I am not sure if it's useful. I wanted looked for a github action that would generate a readable test report.
Because usually it's hard to find which test failed (when a build fails in Github actions).
But this specific action didn't help a lot.

**Pull request merged**